modules/nf-core/gatk4spark/markduplicates/tests/main.nf.test.snap

```diff
@@ -0,0 +1,289 @@
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam.bai:md5,7f7e858d1ded1cca89b373eb817fcb45"
+                    ]
+                ],
+                "metrics": [
+                    
+                ],
+                "output": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam:md5,898cb0a6616897d8ada90bab53bf0837"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,2a299cc62d2a9db49e855f1cc98b2385"
+                ]
+            }
+        ],
+        "timestamp": "2023-12-19T18:14:18.792999573"
+    },
+    "homo_sapiens - bam - multiple": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam:md5,898cb0a6616897d8ada90bab53bf0837"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam.bai:md5,7f7e858d1ded1cca89b373eb817fcb45"
+                    ]
+                ],
+                "2": [
+                    
+                ],
+                "3": [
+                    "versions.yml:md5,2a299cc62d2a9db49e855f1cc98b2385"
+                ],
+                "bam_index": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam.bai:md5,7f7e858d1ded1cca89b373eb817fcb45"
+                    ]
+                ],
+                "metrics": [
+                    
+                ],
+                "output": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam:md5,898cb0a6616897d8ada90bab53bf0837"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,2a299cc62d2a9db49e855f1cc98b2385"
+                ]
+            }
+        ],
+        "timestamp": "2023-12-19T18:14:01.679944552"
+    },
+    "homo_sapiens - cram": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test.bam:md5,898cb0a6616897d8ada90bab53bf0837"
```

Comment:
Why is this a bam, shouldn't it be a cram?

Reply:
yes! thank you for catching.

Reply:
the cram test was pointing to a config but the config had the wrong process selector name in it...it was carried over from the pytests, which allowed aliases. i have removed the alias and now the config is being correctly applied. cram is showing up. let me know if there's anything else!